pykafka/protocol.py

```diff
@@ -262,6 +307,15 @@ def pack_into(self, buff, offset):
         struct.pack_into('!I', buff, offset, crc)
 
     @property
+    def message_offset(self):
```

Comment:
This naming is confusing when looked at alongside `message.offset` and `message.compressed_in_offset`. I actually think the right move here might be to replace both of those properties with this one, if we end up adopting this approach.